gh-actions-scripts/build-helm-charts.sh

```diff
@@ -36,6 +36,9 @@ find . -name values.yaml -exec sed -i -- "s/docker.io\/keptn\//docker.io\/${DOCK
 
 mkdir keptn-charts/
 
+helm repo add bitnami https://charts.bitnami.com/bitnami
```

Comment:
why adding these repos if dependencies are already baked to charts?

Reply:
because we call `helm dependency build` later on. and that command fails if it can't pull the deps